src/utilities/add-remove.test.js

```diff
@@ -0,0 +1,203 @@
+/**
+ * @jest-environment jsdom
+ */
+const { expect, test } = require('@jest/globals');
+const Task = require('./task.js');
+const LocalStorage = require('./localStorage.js');
+
+describe('Test add', () => {
+  // Arrange --------------------------------------------------------------------------->
+  document.body.innerHTML = '<div>'
+     + '<input type="text" name="task" id="task" class="input-task" value="Hello">'
+     + '<ul class="container-list"></ul>'
+     + '</div>';
+  const event = {
+    keyCode: 13,
+  };
+
+  //   class LocalStorage {
+  //     static list = [];
+
+  //     static setDataLocalStorage(item) {
+  //       this.list = item;
+  //     }
+
+  //     static getDataLocalStorage() {
+  //       return this.list;
+  //     }
+  //   }
```

Comment:
- In order to make your code more readable and maintainable please I'd suggest that you remove all commented code from your code base 🙏